validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteBeaconNodeApi.java

```diff
@@ -92,25 +92,19 @@ public static BeaconNodeApi create(
 
     final MetricsSystem metricsSystem = serviceConfig.getMetricsSystem();
 
-    ValidatorApiChannel validatorApi;
-    if (failoverEndpoints.isEmpty()) {
-      validatorApi = primaryValidatorApi;
-    } else {
-      LOG.info(
-          "Will use {} as a primary Beacon Node endpoint and {} as failovers",
-          primaryEndpoint,
-          failoverEndpoints);
-      validatorApi =
-          new FailoverValidatorApiHandler(
-              primaryValidatorApi,
-              failoverValidatorApis,
-              failoversSendSubnetSubscriptions,
-              metricsSystem,
-              ValidatorLogger.VALIDATOR_LOGGER);
+    if (!failoverEndpoints.isEmpty()) {
+      LOG.info("Will use {} as failover Beacon Node endpoints", failoverEndpoints);
```

Comment:
Have these endpoint URLs been sanitised to remove any credentials that may be in them?

Reply:
Yeah, on line 76 there is 
```java
// Strip any authentication info from the URL(s) to ensure it doesn't get logged.
endpoints = stripAuthentication(endpoints);
```